xjalienfs.sh

```diff
@@ -15,11 +15,11 @@ requires:
 env PYTHONUSERBASE="$INSTALLROOT" ALIBUILD=1 pip3 install --user file://${SOURCEDIR}
 XJALIENFS_SITEPACKAGES=$(find ${INSTALLROOT} -name site-packages)
 
-ALIEN_PY=$(find ${INSTALLROOT} -name alien.py)
+ALIEN_PY=$(find ${INSTALLROOT} -path '*/bin/*' -name alien.py)
```

Comment:
How do you guarantee there is only one?

Reply:
Well, we install alien.py using setup.py and pip and there will be just one bin/alien.py for sure. But, I agree, I should change this to an explicit path.

Reply:
ok, but can't you simply use `${INSTALLROOT}/bin/alien.py`, then?